discovery_windows.go

```diff
@@ -53,6 +58,27 @@ func (s *PHPStore) doDiscover() {
 	}
 }
 
+func (s *PHPStore) addFromEnv() {
+
+	// Determine the executable name based on the operating system
+	exeName := "php"
+	if runtime.GOOS == "windows" {
+		exeName = "php.exe"
+	}
+
+	// Alternative approach using exec.LookPath
+	phpPath, err := exec.LookPath(exeName)
```

Comment:
```suggestion
	phpPath, err := exec.LookPath("php.exe")
```

(no need to check for GOOS because we are already only targeting windows in this file